t/t7502-commit-porcelain.sh

```diff
@@ -262,13 +262,57 @@ test_expect_success 'cleanup commit messages (scissors option,-F,-e, scissors on
 
 	echo >>negative &&
 	cat >text <<-\EOF &&
+
 	# ------------------------ >8 ------------------------
 	to be removed
 	EOF
 	git commit --cleanup=scissors -e -F text -a --allow-empty-message &&
 	git cat-file -p HEAD >raw &&
 	sed -e "1,/^\$/d" raw >actual &&
-	test_must_be_empty actual
+	test_must_be_empty actual 
+'
+
+test_expect_success 'cleanup commit messages (scissors option,-F,-e, CR/LF line endings)' '
+
+	echo >>negative &&
+	test_config core.editor unix2dos &&
+	cat >text <<-\EOF &&
+
+	# to be kept
+
+	   # ------------------------ >8 ------------------------
+	# to be kept, too
+	# ------------------------ >8 ------------------------
+	to be removed
+	# ------------------------ >8 ------------------------
+	to be removed, too
+	EOF
+
+	cat >expect <<-\EOF &&
+	# to be kept
+
+	   # ------------------------ >8 ------------------------
+	# to be kept, too
+	EOF
+	git commit --cleanup=scissors -e -F text -a &&
+	git cat-file -p HEAD >raw &&
+	sed -e "1,/^\$/d" raw >actual &&
+	test_cmp expect actual 
+'
+
+test_expect_success 'cleanup commit messages (scissors option,-F,-e, scissors on first line, CR/LF line endings)' '
+
+        echo >>negative &&
+        cat >text <<-\EOF &&
+
+	# ------------------------ >8 ------------------------
```

Comment:
It looks as if the lines above were indented using spaces, but here you use a tab. Git's coding style is to use tabs, always.

I cannot fail to see that you left an empty line after the `test_expect_success` line (which BTW is too long, it has to be at most 80 columns). Both coding style violations are _already_ present in the test script, so I don't fault you for copy/editing those. But I would like to request to fix the style of the lines you added. (You don't need to bother with lines you did not touch, of course, that's not your responsibility.)